scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -0,0 +1,118 @@
+"""Module to generate a column map to their corresponding statistical variable defintions"""
+
+import os
+import json
+import pandas as pd
+
+#TODO:Remove this import and make it from ~/data/utils/ directory when it is available
+os.system('curl https://raw.githubusercontent.com/Abilityguy/data/dcid_gen/util/statvar_dcid_generator.py -LO statvar_dcid_generator.py')
+import statvar_dcid_generator
+
+dcid_gen = statvar_dcid_generator.StatVarDcidGenerator()
+
+
+def generate(specDict, columnList):
+  """Wrapper function for generateColMapBase class to generate column map"""
```

Comment:
For public functions please follow docstring guidelines - https://google.github.io/styleguide/pyguide.html#382-modules

Reply:
got it! I had put a minimal docstring as a placeholder. This shall be updated in the next commit